src/lib/TranscriptEditor/index.js

```diff
@@ -99,25 +117,25 @@ class TranscriptEditor extends React.Component {
     }
 
     this.setCurrentTime(startTime);
-  }
+  };
 
   // eslint-disable-next-line class-methods-use-this
-  handleTimeUpdate = (e) => {
+  handleTimeUpdate = e => {
     const currentTime = e.target.currentTime;
     this.setState({
-      currentTime,
+      currentTime
     });
-  }
+  };
 
-  handlePlayMedia = (bool) => {
+  handlePlayMedia = bool => {
```

Comment:
I must have missed this from before - but can we not do this. bool is the worst name 🙃

Reply:
We've changed this elsewhere I believe. so keep things in line with the rest of the codebase.

https://github.com/bbc/react-transcript-editor/blob/ad9526fba90a3fec5cfaef12b4540238f8f9abbf/src/lib/TranscriptEditor/MediaPlayer/PlayerControls.js#L147

we use `isPlaying` here, and propTypes declares it as a boolean. `bool` isn't useful because the true or false value doesn't describe what state it should take.

Reply:
Sidenote - I think this is bad logic that elsewhere, it's a function: https://github.com/bbc/react-transcript-editor/blob/ad9526fba90a3fec5cfaef12b4540238f8f9abbf/src/lib/TranscriptEditor/TimedTextEditor/index.js#L548

one way or the other. needs some refactoring

Reply:
ok, changed it to `isPlaying`, well spotted.